### PR TITLE
SQL-render friendly statements

### PR DESCRIPTION
**Pull request by Jake-Gillberg**
Changes needed to run the study at Tufts Medical Center

SQL that is closer to conforming to these rules: https://ohdsi.github.io/SqlRender/articles/UsingSqlRender.html#functions-and-structures-supported-by-translate-1

DROP IF EXISTS -> IF OBJECT_ID IS NOT NULL DROP

CREATE TABLE AS -> SELECT INTO

UPDATE table alias FROM join_table -> UPDATE table FROM table INNER JOIN join_table

CEIL -> CEILING

GROUP BY 1 -> GROUP BY coulmn_name

### CODE REVIEW

File: inst/sql/sql_server/quartiles/IQRComplementaryTables.sql [hunk 1]
```diff
@@ -243,30 +247,36 @@ WHERE condition_era_start_date <= cohort.cohort_start_date;
 
 -- Update weights to avoid double counts of mild/severe course of the disease
 -- Diabetes
-UPDATE @cohort_database_schema.charlson_map t1
-SET weight = 0
-FROM @cohort_database_schema.charlson_map t2
-WHERE t1.subject_id = t2.subject_id
-  AND t1.cohort_definition_id = t2.cohort_definition_id
-  AND t1.diag_category_id = 10
+UPDATE t1
```
[the remark this answers is not in the document]
Comment (keesvanbochove) [reply inst/sql/sql_server/quartiles/IQRComplementaryTables.sql:250]: Thanks a lot @Jake-Gillberg for submitting the changes! The changes look fine, I've tested them on Postgres and the only issue is the one I marked above. Are you using SQL Server?

[the remark this answers is not in the document]
Comment (Jake-Gillberg) [reply inst/sql/sql_server/quartiles/IQRComplementaryTables.sql]: Yes, we are running SQL Server. The original query throws the error `Incorrect syntax near 't1'`. Just updated to not use any alias for the updated table. It works on SQL server. Mind pulling the new commit and testing on Postgres?

[the remark this answers is not in the document]
Comment (Jake-Gillberg) [reply inst/sql/sql_server/quartiles/IQRComplementaryTables.sql]: Actually realized that I had a Postgres server lying around. My new update still doesn't work on postgres. I'm not sure without digging further how to write this query to work on both postgres and sql server

[the remark this answers is not in the document]
Comment (Jake-Gillberg) [reply inst/sql/sql_server/quartiles/IQRComplementaryTables.sql]: Postgres docs
> from_list
>
>    A list of table expressions, allowing columns from other tables to appear in the WHERE condition and the update expressions. This is similar to the list of tables that can be specified in the FROM Clause of a SELECT statement. Note that the target table must not appear in the from_list, unless you intend a self-join (in which case it must appear with an alias in the from_list).

SQL Server docs
>FROM <table_source>
>Specifies that a table, view, or derived table source is used to provide the criteria for the update operation. For more information, see FROM (Transact-SQL).
>
>If the object being updated is the same as the object in the FROM clause and there is only one reference to the object in the FROM clause, an object alias may or may not be specified. If the object being updated appears more than one time in the FROM clause, one, and only one, reference to the object must not specify a table alias. All other references to the object in the FROM clause must include an object alias.

[the remark this answers is not in the document]
Comment (Jake-Gillberg) [reply inst/sql/sql_server/quartiles/IQRComplementaryTables.sql]: Okay, most recent commit should resolve this. Did some small tests on Postgres and SQL server. Feels like there should be a better way to write this query..

**Comment by keesvanbochove** [comment]
Thanks, I confirmed that this works in Postgres also! Hopefully it will work in the other databases as well.

**Pull request merged**